### ISSUE TITLE
environment variable to specify sumologic environment

### ISSUE THREAD

**Issue by marionettist** [SumoLogic/sumologic-terraform-provider#12]
Currently, the only way of providing the sumo environment name to use is by specifying it in the environment variable in the provider e..g.
```
provider "sumologic" {
  environment = "${var.sumo_environment}"
}
```
This works well for most cases, but not being able to set it via an environment variable makes running the provider's acceptance tests more difficult for developers with sumo credentials in regions other than the U.S. (when the access keys are associated with a sumo environment such as 'eu')

I will raise a PR for this

**Comment by frankreno** [comment]
will close once we cut a new release.  @marionettist since you have multiple things in the works, I will delay cutting a release for a bit so we can get more things in.

**Comment by frankreno** [comment]
fixed in v0.2.0
